cirq-google/cirq_google/devices/grid_device.py

```diff
@@ -37,6 +37,9 @@
 MEASUREMENT_GATE_FAMILY = cirq.GateFamily(cirq.MeasurementGate)
 WAIT_GATE_FAMILY = cirq.GateFamily(cirq.WaitGate)
 
+# Families of gates which can be applied to any subset of valid qubits.
+_SUBSET_PERMUTATION_GATE_FAMILIES = [MEASUREMENT_GATE_FAMILY, WAIT_GATE_FAMILY]
```

Comment:
Could we perhaps call this `_VARIADIC_GATE_FAMILIES` instead? I think the term "variadic" captures the property we want, that these are gates that can be applied to any number of qubits.

Reply:
I am going to remember this vocabulary word for when I take the Quantum SATs!